tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { compose, createStore, applyMiddleware, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import loggerMiddleware from 'lib/middlewares/loggerMiddleware';
+import reducers from '../reducers';
+import { initalStates } from '../reducers';
+
+export default props => {
```

Comment:
```
Create a function that takes initialProps and returns a store.
```

@alexfedoseev How about we rename the param `initialProps` and rename the `initialName` as `name`.